website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,111 @@
+
+### A language of their own
+The open source community can sometimes be overwhelming with its acronyms and technical terms. Newcomers might find it difficult to keep up with the jargon. However, there are ways to overcome this, such as taking the time to research and ask questions. Also, the Cloud Native Computing Foundation (CNCF) provides a helpful [glossary of commonly used terms](https://glossary.cncf.io/abstraction/) in the cloud native landscape. By utilizing this resource and being proactive in seeking out information, newcomers can better understand the community and its practices. Open source communities aim to be collaborative and inclusive, and its members are generally welcoming to those who are willing to learn and contribute.
+
+### Start Small
+When you're first getting started with open source, it's important to start small. Look for easy issues to fix or small features to add. This will help you get familiar with the project's codebase and build your confidence as a contributor. As you become more experienced, you can take on more challenging tasks.
+
+### Collaborate with Others
+Open source is all about collaboration. Don't be afraid to reach out to other contributors for help or feedback. You can also collaborate with other contributors by reviewing their code, providing feedback, and testing their changes. This will help you build relationships with other contributors and improve your coding or other skills you bring to the community. Contributing to an open source project doesn't require permission. Once you've done sufficient research and have a clear understanding of where your skills can help the community, don't hesitate to dive in and start contributing.
+
+### Asking for help
+
+Getting help from other contributors is a critical aspect of contributing to open source projects, but it's essential to know how much help is appropriate to ask for. It's okay to ask for assistance if you've made a genuine effort to learn the material beforehand, such as reading the documentation or using the software. Try to limit your requests to project-specific issues and avoid seeking help for general mentoring in open source or programming languages. 
+
+While seeking guidance on pull requests can be helpful, it's essential to keep in mind that simply asking for someone to provide you with the exact code or wording to use may not be the most effective approach. Instead, put in effort to create your own content and seek feedback on how you can improve your contribution. This way, you can work collaboratively with other contributors to achieve the project's objectives and build strong relationships within the community. 
+
+Remember, the goal is to help the project and maintainers, so ensure that your requests contribute to their efforts and do not increase their workload. By following these guidelines, you can build strong relationships with other contributors while improving your coding skills.
+
+### Technical Contributors
```

Comment:
Should this use "Code Contributors" here in contrast to using "non-code contributors" below? 🤔

Reply:
Agree, "Code Contributors" is better. updated.